sylvia-derive/src/message.rs

```diff
@@ -209,17 +211,18 @@ impl<'a> EnumMessage<'a> {
             })
             .collect();
 
-        let associated_exec: Option<Type> = source.items.iter().find_map(|item| match item {
-            TraitItem::Type(ty) if ty.ident == "ExecC" => {
-                Some(parse_quote! { <C as #trait_name>::ExecC })
-            }
-            _ => None,
-        });
+        let associated_exec = parse_associated_custom_type(source, "ExecC");
+        let associated_query = parse_associated_custom_type(source, "QueryC");
 
-        let resp_type = match associated_exec {
-            Some(exec) if !custom.has_msg() => exec,
-            _ => custom.msg(),
-        };
+        let resp_type = custom
+            .msg()
+            .or(associated_exec)
+            .unwrap_or_else(Custom::default_type);
```

Comment:
I am wondering (as it is always `Empty` it could be maybe `Custom::empty`), but it is fine (just thinking about naming). Just for consideration, not a change request.